src/Components/Authorization/commonElements.tsx

```diff
@@ -0,0 +1,67 @@
+import React from "react"
+import styled from "styled-components"
+
+import Yup from "yup"
+import FacebookButton from "../Buttons/Facebook"
+import InvertedButton from "../Buttons/Inverted"
+import TwitterButton from "../Buttons/Twitter"
+import Checkbox from "../Checkbox"
+import Input from "../Input"
+
+export const inputValidators = {
+  name: Yup.string().required("Name is required"),
+  password: Yup.string()
+    .required("Password required")
+    .min(8, "Your password must be at least 8 characters"),
+  acceptedTermsOfService: Yup.boolean().required(
+    "You must agree to our terms to continue."
+  ),
+  email: Yup.string()
+    .email("Please enter a valid email.")
+    .required("Please enter a valid email."),
+}
+
+export const FormContainer = styled.form`
+  display: flex;
+  flex-direction: column;
+  width: 400px;
+  height: 500px;
+  padding: 15px 20px;
+`
+
+const buttonWidth = "100%"
+
+export const StyledFacebookButton = styled(FacebookButton)`
```

Comment:
In [the docs](https://www.styled-components.com/docs/basics#extending-styles) they recommend using `.extend` (`FacebookButton.extend`) over the `styled()` HOC.